### DISCUSSION TITLE
Update instructions when using CloudFormation

### DISCUSSION THREAD

**Discussion by mubarak-j** [aws-solutions-library-samples/cloud-intelligence-dashboards-framework#299] [Q&A]
The [workshop docs](https://www.wellarchitectedlabs.com/cost/200_labs/200_cloud_intelligence/cost-usage-report-dashboards/dashboards/2b_cudos_dashboard/#update-dashboard-template---optional) for various dashboards only include `cid-cmd` and manual update instructions. What are the update instructions when using cloudformation? do I simply grab the recent version from this repo and upload it to CF?

**Comment by iakov-gan** [comment]
Close cleanup. Please comment if still an issue
